openmetadata-ui/src/main/resources/ui/src/pages/TestSuitePage/TestSuitePage.tsx

```diff
@@ -269,6 +316,18 @@ const TestSuitePage = () => {
             </Col>
           )}
         </Row>
+        <DeleteWidgetModal
+          afterDeleteAction={fetchTestSuites}
+          allowSoftDelete={!showDeleted}
+          entityId={selectedTestSuite?.id || ''}
+          entityName={selectedTestSuite?.name || ''}
+          entityType="testSuite"
+          prepareType={false}
+          visible={!isUndefined(selectedTestSuite)}
+          onCancel={() => {
+            setSelectedTestSuite(undefined);
+          }}
+        />
```

Comment:
can you try this with the latest main? 
- merge main in your PR
- re-build
- and try this action
API will fail, we need to remove `prepareType={false}`, after that API will pass
ref- https://github.com/open-metadata/OpenMetadata/issues/10041

Reply:
Ok I'll try that